optical/converter/converter.py

```diff
@@ -300,6 +300,103 @@ def convert_coco(
             _fastcopy(src_dir, dest_dir, images)
 
 
+def _make_manifest_data(image_info: List, grouped_info: pd.DataFrame, job_name: str, id_to_class_map: Dict):
+    # creating json like data for each row of df
+    manifest_dic = {}
+    manifest_dic["source-ref"] = image_info[0]
+    manifest_dic[f"{job_name}"] = {
+        "image_size": [{"width": int(image_info[1]), "depth": 3, "height": int(image_info[2])}],
+    }
+    manifest_dic[f"{job_name}-metadata"] = {
+        "job-name": job_name,
+        "class-map": id_to_class_map,
+        "creation-date": str(datetime.now()),
+        "type": "groundtruth/object-detection",
+    }
+    annotations = []
```

Comment:
`iterrows` is pretty slow. Please see if you can come up with an alternative, e.g. something similar to 
`list(df.to_dict(orient="index").values())`?